contributors/design-proposals/selector-generation-from-kubectl.md

```diff
@@ -0,0 +1,174 @@
+Selector generation from kubectl
+==================
+
+# Goals
+
+Make `kubectl apply` work well with selector.
```

Comment:
I'm inclined to say that everything should be using generated selectors, by default. With the ability to turn it off by a user, when needed. Selectors are for controllers to work with underlying resources, usually. There's nothing stooping users from adding additional labels, if needed.